spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+feature 'comments' do
+  let!(:pfc) {Picture.create(title:'PFC',
+                             image: File.new("spec/features/Pompey.jpg") )}
```

Comment:
Expression at 5, 75 should be on its own line.<br>Space inside parentheses detected.